pitest-maven/src/main/java/org/pitest/maven/report/generator/XMLReportGenerator.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015 Jason Fehr
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+package org.pitest.maven.report.generator;
+
+
+public class XMLReportGenerator implements IReportGenerationStrategy {
```

Comment:
By copying over the pit html report, the maven site opens it in a new window instead of within the context of the maven site (i.e. so the site header and left-side navigation are not visible).  I was hoping to to use the doxia sink to generate a more integrated report from the pit xml data file but did not have the time right now.  My line of thought was to build it later or have someone else build it.

I will definitely drop the class if that is the better design.  It would be very easy to recreate later.